document_splitter/src/get_top_samples.py

```diff
@@ -87,6 +89,7 @@ def get_all_scores(
     scores = pd.DataFrame(columns=data_df.columns.tolist())
     for i in range(0, len(data_df)):
         scores.at[i, ColumnNames.IMAGE_NAME.value] = data_df.at[i, ColumnNames.IMAGE_NAME.value]
+        scores.at[i, ColumnNames.PAGE_NUM.value] = data_df.at[i, ColumnNames.PAGE_NUM.value]
 
         # document encoding depends on whether data_distinction_type is image or text.
         if data_distinction_type == DataType.IMAGE.value:
```

Comment:
1 - what if data_distinction_type is passed 'Image'? - Your code would encode it as a text, because it doesn't match DataType.IMAGE.value. I would use elif for text and else for throwing exceptions that data_distinction_type is not properly specified.
2 - This code can be extracted as a method itself. This would make your function more readable.
3 - Name xtest doesn't say much. Please pick a better name

Reply:
Link - https://github.com/SalesHero-Inc/hero-document-clustering/pull/2/files#diff-e360391dec0505ac18d53f09e41e55343f6f0615530543597677eab7afb7f490R64